services/net/filemonitor/appsettings.json

```diff
@@ -11,7 +11,7 @@
   "Service": {
     "MaxFailLimit": 5,
     "ApiUrl": "http://api:8080",
-    "VolumePath": "/data",
+    "VolumePath": "/workspaces/tno/monitored",
```

Comment:
This has been a habit of mine since I was testing the capture and clip services together. I needed a location that was accessible by the dev containers of both services. I also didn't like the restrictions on accessing the container's root directory. The issue here is that I would forget to set the VolumePath back to /data before I pushed, which is what happened here. As the filemonitor doesn't share its data with any other service, I created /data/papers in the dev container and made it accessible by the service.